app/modules/inventory/components/item_row.svelte

```diff
@@ -33,16 +35,38 @@
 <div class="item-row">
   <slot name="checkbox" />
 
-  <a href="{pathname}" on:click|stopPropagation={loadInternalLink}>
-    <div class="image-wrapper">
-      {#if image}<img src="{imgSrc(image, 128)}" alt="">{/if}
-    </div>
-    <div class="info">
-      <p class="title">{title}</p>
-      <p class="authors">{authors || ''}</p>
-    </div>
-    <p class="details">{details}</p>
-  </a>
+  <div class="middle">
+    <a
+      href="{pathname}"
+      on:click|stopPropagation={loadInternalLink}
+      class="show-item"
+    >
+      <ImageDiv url={image} size={128} />
+      <div class="info">
+        <p class="title">{title}</p>
+        <p class="authors">{authors || ''}</p>
+        {#if $screen.isSmallerThan('$smaller-screen')}
+          <p class="details">{details}</p>
+        {/if}
+      </div>
+      {#if $screen.isLargerThan('$smaller-screen')}
+        <p class="details">{details}</p>
+      {/if}
+    </a>
+
+
+    {#if showUser && $itemStore.user}
+      {@const { username, picture, inventoryPathname } = $itemStore.user}
```

Comment:
im not sure why this is necessary, why not extracting `user` within reactive block line 22 ?

Reply:
done in 8ac08b2